nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/appearance/bigIcon.png.mimetype

```diff
@@ -0,0 +1 @@
+image/png
```

Comment:
Could you make it quadratic and with a transparent background (only outside of the Q)?